uax29/Tokenizer.cs

```diff
@@ -94,19 +94,9 @@ public static StreamTokenizer<char> Create(TextReader stream, TokenType tokenTyp
 		return new StreamTokenizer<char>(buffer, tok);
 	}
 
-	static readonly Dictionary<TokenType, Split<byte>> byteSplits = new()
-	{
-		{TokenType.Words, Words.SplitUtf8Bytes},
-		{TokenType.Graphemes, Graphemes.SplitUtf8Bytes},
-		{TokenType.Sentences, Sentences.SplitUtf8Bytes},
-	};
+	static readonly Split<byte>[] byteSplits = [Words.SplitUtf8Bytes, Graphemes.SplitUtf8Bytes, Sentences.SplitUtf8Bytes];

```

Comment:
Nice. Smaller!

Reply:
Faster too, just a bounds check + a load.

Reply:
If you’re so inclined, wanna put in a PR just for this? Or I’ll just copy-pasta.